### PR TITLE
Fixed warnings in akka-bench-jmh-typed

### PR DESCRIPTION
**Pull request by helena**

### CODE REVIEW

File: akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala [hunk 1]
```diff
@@ -143,6 +145,7 @@ object TypedBenchmarkActors {
     }
   }
 
+  @silent("deprecated")
```
Comment (chbatey) [inline akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala]: what is deprecated here?

Comment (helena) [reply akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala]: @chbatey `val` keyword in for comprehension
```
/akka/akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala:149:19: 
val keyword in for comprehension is deprecated
[error]       val message = Message(pong) // just allocate once
```

File: akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala [hunk 2]
```diff
@@ -66,7 +68,7 @@ object TypedBenchmarkActors {
       numActors: Int,
       dispatcher: String,
       batchSize: Int,
-      shutdownTimeout: FiniteDuration): Behavior[Start] =
+      @unused shutdownTimeout: FiniteDuration): Behavior[Start] =
```
Comment (raboof) [inline akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala]: can/should we just remove it then?

**Comment by helena** [comment]
Review suggestions pushed

**Review by raboof — APPROVED** [review]

File: akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala (hunk 1 above)
Comment (chbatey) [reply akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala]: Can we fix it then? Just remove the val?

**Review by chbatey — APPROVED** [review]
LGTM once val is removed

**Pull request merged**